CRM/Speakcivi/Tools/Dictionary.php

```diff
@@ -453,4 +453,40 @@ public static function getMessageContent($filename, $default) {
     }
     return $content;
   }
+
+  public static function fallbackCityValues() {
+    return array(
+        "fr_FR" => "votre ville",
```

Comment:
You are probably tracking this with Anne, but just so it doesn't get missed: it, es and pt (and maybe nl) are missing from the list.

Reply:
Yep - I'll update the list with whatever languages actually work.